src/needsahome.jl

```diff
@@ -25,7 +25,7 @@ function buildSubgraphFromLabels!(dfg::G,
                                   syms::Vector{Symbol};
                                   subfg::AbstractDFG=(G <: InMemoryDFGTypes ? G : GraphsDFG)(params=getSolverParams(dfg)),
```

Comment:
Note GraphsDFG is hard coded here. Depending on the IIF use case, you might have to change it here if the default is changed in IIF.

Reply:
Yeah, think you defined a more standard `InMemDFGType` -- can we just use that?

Reply:
see #266